test/dhcp_isc/integration_test.rb

```diff
@@ -39,3 +40,22 @@ def test_features
     end
   end
 end
+
+class DhcpApiValidIPTest < Test::Unit::TestCase
+  include Rack::Test::Methods
+
+  def app
+    app = Proxy::DhcpApi.new
+    app.helpers.server = @server
+    app
+  end
+
+  def test_validate_ip
+    @server = mock()
+    @server.stubs(:unused_ip).returns(true)
```

Comment:
This is still a good idea, but I'd rather stub this on the actual implementation.